CodeForPoznan/pah-fm
# [321] Show localized warning to the user when drive is not verified

Closes #321 :dancer: 

How to test it:
make start
submit drive without filling in signature field (the last one)
notice the warning is shown in form view and in drives list view
